README.md

```diff
@@ -1,5 +1,24 @@
 #  Under construction
-
+Please note that current project is in active development and everything here is a subject to change.
 Currently migrating code from other sources.
-
 ![alt text](assets/sign.png "Construction sign")
+
+
+
+# Prerequisites
+You will need this packages to be installed:
```

Comment:
Outside of pytorch neither of these are strictly required. torchvision is never imported and ipython + notebook is only needed for the examples.

Reply:
It wont compile without torchvision. 
Usually prerequisites list everything that you need to have the whole project working and not just some parts?